assets/html/volunteer.html

```diff
@@ -0,0 +1,373 @@
+            <span class="tooltiptext">Medium</span>
+          </a>
+        </div>
+      </div>
+    </header>
+
+    <div class="small-hero" id="volunteer">
+      <h1>Volunteer</h1>
+      <div id="artist-info">
+        <p>Jonny B. Artist</p>
+        <p><a href="#">@some-social-handle</a></p>
+      </div>
+    </div>
+    <div class="h2bar"></div>
+    <h2>Getting Started</h2>
+    <div class="text-content">
+      <p class="notify">In response to the Covid-19 pandemic, we are currently holding hack nights virtually. Please see
+        our
+        Meetup page or join our Slack channel for details and registration links.</p>
+      <ol id="gettingStartedList">
+        <li>
+          <div class="getting_started">
+            <a href="https://slackin-c4sj.herokuapp.com/" target="_blank" rel="noopener noref">
+              <p>Join Our Slack</p>
+              <img src="../img/social_logos/Slack_RGB_White.png" alt="Slack_Mark.svg" />
+            </a>
+          </div>
+        </li>
+        <li>
+          <div class="getting_started">
+            <a href="https://www.meetup.com/Code-for-San-Jose/" target="_blank" rel="noopener noref">
+              <p>Attend a Meetup</p>
+              <img src="../img/social_logos/meetup-logo-script.png" alt="meetup-logo-m-swarm.svg" />
+            </a>
+          </div>
+        </li>
+        <li>
+          <div class="getting_started">
+            <a href="https://github.com/codeforsanjose/" target="_blank" rel="noopener noref">
+              <p>Browse Our Github</p>
+              <img src="../img/social_logos/GitHub_Logo_White.png" alt="GitHub-Mark-Light-120px-plus.png" />
+            </a>
+          </div>
+        </li>
+      </ol>
+      <p>
+        Our online Slack chat group and in-person Meetups are great places to
+        get started helping with a project! Anyone can join! Technical skills
+        are not required to contribute.
+      </p>
+      <p>
+        <strong>1. Join Our Slack Chat</strong>
+        <a href="https://slackin-c4sj.herokuapp.com/" target="_blank" rel="noopener noref"><button
+            class="volunteer-btns">JOIN
+            SLACK</button></a>
+      </p>
+      <p>
+        We use Slack, an online chat platform, as our primary communication
+        channel to collaborate on projects and stay updated on events.
+      </p>
+      <p>Sign up to our Slack chat group and introduce yourself!</p>
+      <p>
+        <strong>2. Attend a Meetup</strong>
+        <a href="https://www.meetup.com/Code-for-San-Jose/" target="_blank" rel="noopener noref"><button
+            class="volunteer-btns">SEE MEETUPS</button></a>
+      </p>
+      <p>
+        Our next in-person event details are available on our Meetup page. We
+        onboard new members at meetings. You can also meet our existing
+        members and team leads. Everyone is welcome.
+      </p>
+      <p>
+        Code for San José typically meets two Thursdays a month at 6:30pm at
```

Comment:
Until further notice, our meetups will be held virtually. Details on Meteup.

Reply:
I added the bold sections in red to indicate meeting virtually as well.

Reply:
Sentence changed.